tmt/utils.py

```diff
@@ -653,6 +660,22 @@ def environment_to_dict(variables):
     return result
 
 
+def context_to_dict(context):
+    """
+    Convert command line context definition into a dictionary
+
+    Does the same as environment_to_dict() plus separates possible
+    comma-separated values into lists. Here's a couple of examples:
+
+    distro=fedora-33 ---> {'distro': ['fedora']}
+    arch=x86_64,ppc64 ---> {'arch': ['x86_64', 'ppc64']}
+    """
+    result = dict()
```

Comment:
Can be written with dict comprehension as 

```
return {
    key: value.split(',') for key, value in environment_to_dict(context).items()
}
```

right?

Reply:
Nice hint, updated, thanks.